packages/components/cascader-panel/src/index.vue

```diff
@@ -75,13 +75,13 @@ export default defineComponent({
   emits: [UPDATE_MODEL_EVENT, CHANGE_EVENT, 'close', 'expand-change'],
 
   setup(props, { emit, slots }) {
-    let initialLoaded = true
     // for interrupt sync check status in lazy mode
     let manualChecked = false
 
     const config = useCascaderConfig(props)
 
     let store: Nullable<Store> = null
+    const initialLoaded = ref(true)
```

Comment:
I wonder should we rename the variable since it was used for more than indicating initial loading state.

Reply:
I have no idea about renaming this 🤔    
This variable indicates the loading state so I add a UI to tell the user "the data is loading", I don't know how to name it after I added this feature.

Reply:
Let's just call it that way for now. We can figure that out later.